src/__tests__/query.test.js

```diff
@@ -14,48 +16,48 @@ const query = gql`
     }
   }
 `
+
+let uri = 'https://web-app-client-b69l4yjrq-bacpacs-projects.vercel.app/api/graphql'
+console.log('graphql_uri', uri)
+const client = new ApolloClient({
+  link: new HttpLink({ uri, fetch }),
+  cache: new InMemoryCache(),
+})
 async function fetchData() {
   try {
     const result = await client.query({ query })
```

Comment:
Instead of using a try-catch, we can use this statement directly inside the tests. If we do that, then the tests will fail, if the fetching fails.